sdk/messaging/azservicebus/internal/retrier.go

```diff
@@ -130,3 +133,136 @@ func (rp *backoffRetrier) Try(ctx context.Context) bool {
 		return false
 	}
 }
+
+type RetryFnArgs struct {
+	I int32
+	// LastErr is the returned error from the previous loop.
+	// If you have potentially expensive
```

Comment:
Missing some comment here?

Reply:
(not ignoring this, will come back in the next PR and take a look)